src/molecules/List/List.tsx

```diff
@@ -29,16 +29,53 @@ const ListItem = styled(Typography)`
   list-style-position: inside;
 `;
 
+const DefinitionEntry = styled.dl`
+  display: flex;
+  align-items: center;
+  margin-bottom: 1.4375em;
+`;
+
+const DefinitionLabel = styled(Typography)`
+  flex: 1 0;
+  margin: 0;
+  color: ${props => props.theme.cardText};
+  letter-spacing: 0.106875em;
+  text-transform: uppercase;
+  font-weight: bold;
+`;
+
+DefinitionLabel.defaultProps = {
+  as: 'dt',
+};
+
+const DefinitionValue = styled(Typography)`
+  flex: 1 0;
+  margin: 0;
+`;
+
+DefinitionValue.defaultProps = {
+  as: 'dd',
+};
+
+interface Description {
+  term: string;
+  definition: string | ReactNode;
+}
+
 export const List = ({
   basic,
   children,
+  description,
+  descriptionData,
   group,
   inline,
   ordered,
   ...rest
 }: {
   basic?: boolean;
   children: ReactNode;
+  description?: boolean;
```

Comment:
I think we can remove this, since we can't show a description list without `descriptionData` anyway